main_server/local_server/server_admin/server_admin_javascript.js

```diff
@@ -2967,23 +2967,47 @@ function BMLT_Server_Admin()
                 format_keys[0] = parseInt(format_keys_string);
             };
         };
-        
+
+        var venue_type_format_selections = { "HY": false, "TC": false, "VM": false };
         var main_formats = g_format_object_array;
         for (var c = 0; c < main_formats.length; c++) {
             var format_checkbox = document.getElementById('bmlt_admin_meeting_' + in_meeting_object.id_bigint + '_format_' + main_formats[c].id + '_checkbox');
-            
+
+            if (this.isVenueTypeFormatBySharedId(main_formats[c].id)) {
+                $(format_checkbox).closest("div.bmlt_admin_meeting_one_format_div").hide();
+            }
+
             if ( format_checkbox ) {
                 for (var i = 0; i < format_keys.length; i++) {
                     if ( format_checkbox.value == format_keys[i] ) {
+
+                        if (parseInt(this.venueTypeFormats['HY']['shared_id']) === main_formats[c].id) {
+                            venue_type_format_selections['HY'] = true;
+                        } else if (parseInt(this.venueTypeFormats['TC']['shared_id']) === main_formats[c].id) {
+                            venue_type_format_selections['TC'] = true;
+                        } else if (parseInt(this.venueTypeFormats['VM']['shared_id']) === main_formats[c].id) {
+                            venue_type_format_selections['VM'] = true;
+                        }
+
                         format_checkbox.checked = true;
                         break;
                     };
                 };
             };
         };
-        
+
```

Comment:
What was the final decision on what you want implemented?

Reply:
> > > > Overall, I think this approach looks really good. I think I agree with Alan's comment, and made a suggestion for how to move forward. If we decide to move forward with the suggestion I made, I think the only thing missing is some input validation requiring the user to select at least one radio button before saving.
> > > 
> > > 
> > > A radio button is always selected currently.  Because if none of the 3 combinations are valid, `In-Person` is automatically selected.  There is no way to unselect here.
> > 
> > 
> > Right. I made a suggestion in another comment suggesting an alternate implementation, and I provided code. The only thing missing from my example code is the extra input validation to require a selection.
> 
> I'm not following. Unless I'm missing something, there would never be a scenario where a radio button would be unselected, because an option is always pre-selected and calculated.

https://github.com/bmlt-enabled/bmlt-root-server/pull/370#discussion_r549157743

The code here will not always pre-select a radio button.

Reply:
I think so? I don't know if the code works or not. The outcome should be that when a supported combination of formats is not selected, no radio button is selected. Then, the user should not be able to save without making a selection, forcing them to fix their data before they can edit their meeting.